ehabi32/ehabi32.rst

```diff
@@ -2623,6 +2632,14 @@ The encoding has been designed to meet the following criteria.
   f) If a Reserved or Spare code is encountered, the PR will return
      \_URC\_FAILURE.
 
+  g) Functions, compiled with Return Address Authentication will need to keep a
```

Comment:
I think the placement of the commas makes the sentence less readable.

Perhaps: `Functions, compiled with Return Address Authentication, will need to keep a
     pointer authentication code used to validate integrity of the return
     address upon function exit.`